.github/ISSUE_TEMPLATE/plugin-ownership-transfer.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Plugin Ownership Transfer
+about: Propose a plugin ownership transfer to the spinnaker-plugins organization. For more information, please see the documentation for [community donated plugins](https://github.com/spinnaker/governance/blob/master/sig-platform/community-donated-plugins.md)
+title: 'Plugin Transfer: <plugin name>'
+labels: committee/technical-oversight
+assignees: ajordens, ethanfrogers, ezimanyi, plumpy, robzienert
+
+---
+
+### Plugin Information
+
+Name of the plugin.
+
+Description of what the plugin is.
+
+### Acknowledgements
+
+- [ ] Source code will be available publicly on [`spinnaker-plugins`](https://github.com/spinnaker-plugins)
+- [ ] All authors listed below will be given `write` permissions to the plugin repository
+- [ ] Plugin authors are expected to continue as primary maintainers of plugin
+- [ ] Spinnaker core contributors are not obligated to help maintenance of plugin
```

Comment:
"obligated" is quite a strong term - are they perhaps "expected" to help maintain the plugins? Trying to understand the level of involvement we're aiming for here

Reply:
I'm not sure there are or should be any expectations that Spinnaker approvers maintain donated community plugins. 

Worth clarifying but I'm assuming Spinnaker approvers would not automatically inherit similar permissions for the `spinnaker-plugins` organization?

Reply:
My intent with this statement was to convey that the Spinnaker core contributors will _not_ be involved in maintenance of the plugins. The responsibility and duties of code ownership will continue to lie entirely with the original authors. Any word smithing help would be great!

> Worth clarifying but I'm assuming Spinnaker approvers would not automatically inherit similar permissions for the spinnaker-plugins organization?

Good clarification, yes, I'll add that - approvers / reviewers will not have their access applied to plugin repos unless the plugin authors want that.